server/src/ai/ai.functions.ts

```diff
@@ -40,6 +40,50 @@ async function makeSummary(text: string) {
   return completion.choices[0].message.content;
 }
 
+/** generateTags. 
+ * 
+ * Takes in all reviews from a course as text and 
+ * generates 5 tags for those reviews describing lectures, assignments, professor,
+ * skill, and resources as well as the corresponding connotation of that tag.
+ * @params a string that combines all reviews from a course
+ * @returns an array of adjectives and their corresponding connotations describing
+ * lectures, assignments, professr, skills, and resources in that order
+ */
+async function generateTags(text: string) {
+  const completion = await openai.chat.completions.create({
+    model: "gpt-3.5-turbo",
+    messages: [
+      {
+        role: "system", content: `
+          You are creating 5 adjectives describing the lectures, assignments, professor, skills, and resources,
```

Comment:
Should we tell it that it is given a string of consecutive reviews for a single course?